service/frontend/templates/clusterredirection.tmpl

```diff
@@ -104,9 +106,17 @@ func (handler *clusterRedirectionHandler) {{$method.Declaration}} {
 		return nil, err
 		{{- end}}
 	}
+
+	var actClSelPolicyForNewWF *types.ActiveClusterSelectionPolicy
+	var workflowExecution *types.WorkflowExecution
```

Comment:
These are only relevant for active-active domains. Requests for other domain types will still use domainEntry.ActiveClusterName to determine cluster.